dev/tests/static/testsuite/Magento/Test/Php/_files/blacklist/strict_type.txt

```diff
@@ -0,0 +1 @@
+# Format: <componentType=module|library|theme|language|*> <componentName> <globPattern> or simply <globPattern>
```

Comment:
Please reflect these changes in PR description

Reply:
Done.